streams/src/test/java/org/apache/kafka/streams/processor/internals/DefaultStateUpdaterTest.java

```diff
@@ -1139,8 +1209,6 @@ public void shouldGetTasksFromUpdatingTasks() throws Exception {
         stateUpdater.add(standbyTask3);
         verifyUpdatingTasks(activeTask1, activeTask2, standbyTask1, standbyTask2, standbyTask3);
 
-        final Set<Task> tasks = stateUpdater.getTasks();
-
```

Comment:
This is actually not needed since we verify the tasks in `verifyGetTasks()` below.